index.d.ts

```diff
@@ -342,49 +329,58 @@ declare namespace _ReactPixi {
        * Destroy instance on unmount?
        * @default true
        */
-      destroy?: boolean;
+      destroy?: boolean
 
       /**
        * Destroy child instances?
        * @default true
        */
       destroyChildren?: boolean
-    };
+    }
   }
 }
 
 // components
-export const Text: React.FC<_ReactPixi.IText>;
-export const Sprite: React.FC<_ReactPixi.ISprite>;
-export const Container: React.FC<_ReactPixi.IContainer>;
-export const Graphics: React.FC<_ReactPixi.IGraphics>;
-export const BitmapText: React.FC<_ReactPixi.IBitmapText>;
-export const NineSlicePlane: React.FC<_ReactPixi.INineSlicePlane>;
-export const ParticleContainer: React.FC<_ReactPixi.IParticleContainer>;
-export const TilingSprite: React.FC<_ReactPixi.ITilingSprite>;
-export const SimpleRope: React.FC<_ReactPixi.ISimpleRope>;
-export const SimpleMesh: React.FC<_ReactPixi.ISimpleMesh>;
-export const AnimatedSprite: React.FC<_ReactPixi.IAnimatedSprite>;
+export const Text: React.FC<_ReactPixi.IText>
+export const Sprite: React.FC<_ReactPixi.ISprite>
+export const Container: React.FC<_ReactPixi.IContainer>
```

Comment:
```suggestion
export const Container: React.FC<React.PropsWithChildren<_ReactPixi.IContainer>>
```
Since React18, FC no longer includes children, Maybe we need PropsWithChildren? other components as well.

Reply:
@ryohey yeah, you're right, thanks for the suggestion!

Reply:
Fixed